tests/components/Filters/Facets.stories.tsx

```diff
@@ -5,13 +5,12 @@ import { AnswersHeadlessContext, State } from '@yext/answers-headless-react';
 import { generateMockedHeadless } from '../../__fixtures__/answers-headless';
 import { RecursivePartial } from '../../__utils__/mocks';
 import { DisplayableFacets } from '../../__fixtures__/data/filters';
-import { Filters } from '../../../src/components';
-import { Facets } from '../../__compound-components__/Filters/Facets';
+import { Facets, FacetsProps } from '../../../src';
```

Comment:
I think now we can move the Facets stories and jest test files out of the Filters directory to be at the same level as StaticFilters